Pipfile

```diff
@@ -24,6 +24,7 @@ numpy = "*"
 google-cloud-pubsub = "*"
 flask-sqlalchemy = "*"
 flask = "*"
+jinja2 = "*"
```

Comment:
We may do, but did you mean to add this as an explicit dependency? It's a third party dependency of `Flask-Cors`, `Flask-RESTful` and `flask-sqlalchemy`. I'm never sure if we should just upgrade those and let them resolve it.

Reply:
This has been removed. I never meant to have included it. I was just testing a theory.